src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random;
+    private String[] simpleColors = {
+            "Red", "Green", "Blue", "Yellow", "Orange", "Purple", "Pink", "Cyan"
+    };
+
+    public ColorSupplier() {
+        random = new Random();
```

Comment:
It is redundant, remove that constructor and make initialization in that way:
`private final Random random = new Random();`